setup.md

```diff
@@ -4,166 +4,241 @@ This document will guide you through setting up Fennel on your
-to download Fennel at all.
+**Tip**: If you are using software that supports Fennel, such as
+[TIC-80](https://tic.computer), you do not need to download Fennel,
+because you can use it inside of TIC-80.
+
+## Using Git to download and install Fennel
+
+Downloading and installing Fennel using Git allows you to use versions
+of Fennel that haven't been released yet and makes contributions to
+Fennel easier.
 
-## Using Git to download Fennel
+### To download Fennel
 
-You may want to use Git to install if:
+1. `cd` to a directory in which you want to download Fennel, such as
+   `~/src`
+2. Run `git clone https://github.com/bakpakin/Fennel`
 
-* You want to use a version that hasn't been released yet
-* You want to contribute changes to Fennel
-* You have Git installed and don't want to bother with another system
+### To install Fennel
 
-To install:
+1. Run `cd Fennel`
+2. Run `make fennel`
+3. Copy or link `fennel` to a location on your shell's `$PATH`, such
+   as `~/bin` or `/usr/local/bin`
 
-1. Change to the directory where you keep source checkouts (eg `~/src`)
-2. Run `git clone https://github.com/bakpakin/Fennel` in a shell
-3. Change directories with `cd Fennel`
-4. Run `make fennel` to compile Fennel into a standalone script
-5. Copy or link `fennel` to a location on your shell's `$PATH` (eg
-   `~/bin` or `/usr/local/bin`)
+**Note 1**: `Step 2.` above will compile Fennel into a standalone script
+called `fennel`.
 
-## Using LuaRocks to download Fennel
+**Note 2**: If the `fennel` script exists inside your `$PATH`
+directory, you can run `fennel filename.fnl` to run a Fennel file.
+
+## Using LuaRocks to download and install Fennel
 
 [LuaRocks](https://luarocks.org/) contains a repository of Lua
-packages. It allows you to download and install packages and automates
-installation steps.
+software packages. LuaRocks is convenient because it automates the
+downloading, installation, and uninstallation of software packages.
+
+### To download and install Fennel
 
-1. Open up a shell
-2. Ensure the `luarocks` package is installed on your system
-3. Ensure the `~/.luarocks/bin` directory is added to your shell's `$PATH`
-4. Run `luarocks --local install fennel`
+1. Ensure the `luarocks` package is installed on your system
+2. Ensure the `~/.luarocks/bin` directory is added to your shell's `$PATH`
+3. Run `luarocks --local install fennel`
 
-You can try running `fennel --help` to confirm the installation succeeded.
+**Note**: You can try running `fennel --help` to confirm the
+installation succeeded.
 
 # Embedding Fennel
 
-Installing Fennel on your system will allow you to run scripts written
-in pure Fennel. However, for more complex situations it's common to
-embed Fennel code inside a larger application. You can do this in two
-ways: including the Fennel compiler or performing ahead-of-time compilation.
+Fennel can be embedded inside of Lua applications by including the
```

Comment:
Let's say `Fennel code can be embedded...` instead of `Fennel can be embedded`, because that makes it sound like you are putting Fennel (the compiler) in your program, but is not the case when you do AOT compilation.

Reply:
Good point! I'll do that!